app/models/katello/glue/pulp/repo.rb

```diff
@@ -429,8 +429,8 @@ def puppet_module_ids
         Katello.pulp_server.extensions.repository.puppet_module_ids(self.pulp_id)
       end
 
-      def docker_image_ids
-        Katello.pulp_server.extensions.repository.docker_image_ids(self.pulp_id)
+      def docker_img_ids
```

Comment:
@ehelms http://edgeguides.rubyonrails.org/association_basics.html#has-many-association-reference scroll to 4.3.1.6 I am not sure if this was able to be overridden in rails 3 or just didn't exist.